src/ES6/Modules2.js

```diff
@@ -0,0 +1,12 @@
+// const user = require('./Modules.js');
+
+// console.log(`User: ${user.getName()}`);
+
+//↑ This example will work because it use common JS
+
+
+// ↓ This example will  not work if you use common JS so you need to  install typescript
+
+import User from './Modules.js'; // not {User}, just User
+
+new User('John');//<------This ex
```

Comment:
What does `This ex` mean?

Reply:
it was a comment to  say this example but I will delete it